must-gather/collection-scripts/gather

```diff
@@ -91,7 +91,7 @@ else
   log "Skipping collection inspection.  No default CLO found" 2>&1 | tee "${LOGFILE_PATH}"
 fi
 
-loki_crd=$(oc get crd -A -o custom-columns=:.metadata.name | grep lokistack)||
+loki_crd=$(oc get crd -A -o custom-columns=:.metadata.name | grep lokistack)&&
```

Comment:
I believe the correct fix is to have two separate lines and no logical operator or list.

Reply:
I agree with your point. It seems I misunderstood the significance of this logical operator. I will fix it into two separate lines and request a review again.